internal/services/relay/relay_hybrid_connection_resource.go

```diff
@@ -99,7 +99,7 @@ func resourceArmRelayHybridConnectionCreateUpdate(d *pluginsdk.ResourceData, met
 	}
 
 	if _, err := client.CreateOrUpdate(ctx, id, parameters); err != nil {
-		return fmt.Errorf("creating/updating Hybrid Connection %q (Namespace %q Resource Group %q): %+v", id.Name, id.NamespaceName, id.ResourceGroup, err)
+		return fmt.Errorf("creating/updating Hybrid Connection %s: %+v", id, err)
```

Comment:
fwiw this can become:

```suggestion
		return fmt.Errorf("creating/updating %s: %+v", id, err)
```